ocs_ci/ocs/resources/pvc.py

```diff
@@ -413,7 +413,7 @@ def create_restore_pvc(
 
 
 def create_pvc_clone(
-    sc_name, parent_pvc, clone_yaml, pvc_name=None, do_reload=True
+    sc_name, parent_pvc, clone_yaml, pvc_name=None, do_reload=True, storage_size=None
```

Comment:
`storage_size` should not be optional, isn't it?

Reply:
It should be optional to my opinion,since we don't want all our other tests to start failing. And if storage_size is not passed - the the hardcoded value in pvc_clone.yaml will be used ( 1Gi)